libevmasm/Assembly.h

```diff
@@ -47,8 +47,8 @@ class Assembly
 public:
 	Assembly() {}
 
-	AssemblyItem newTag() { return AssemblyItem(Tag, m_usedTags++); }
-	AssemblyItem newPushTag() { return AssemblyItem(PushTag, m_usedTags++); }
+	AssemblyItem newTag() { assertThrow(m_usedTags < 0xffffffff, AssemblyException, ""); return AssemblyItem(Tag, m_usedTags++); }
+	AssemblyItem newPushTag() { assertThrow(m_usedTags < 0xffffffff, AssemblyException, ""); return AssemblyItem(PushTag, m_usedTags++); }
```

Comment:
Limit to be consistent across 32 and 64-bit machines. (`m_usedTags` in `unsigned` and not `size_t`).

I don't think its worth creating a test, which would use more than 4 GB of memory to run.